test.hack

```diff
@@ -0,0 +1,13 @@
+namespace Hack\GettingStarted\MyFirstProgram;
+namespace HH\Lib\Str;
```

Comment:
This file is declared in two namespaces. The MyFirstProgram and Str namespace.

Reply:
good feedback! was just messing around and accidentally added when I wasn't paying attention -- removed.